tracing with jaeger and otlp (#316)
digicatapult/sqnc-ipfs: README.md

Manages a [kubo](https://docs.ipfs.tech/install/command-line/) instance maintaining the private network swarm key based on the value from a `sqnc-node` instance.

## Local development

> install dependencies

```sh
npm i
```

> start substrate node

```sh
docker compose up -d // -d for silent
```

(if you are getting errors like:

```sh
5:20:59 ⚠️  SQNC_IPFS IPFS: Error: no IPFS repo found in /Users/hadamkova/eng/sqnc/sqnc-ipfs/data.
15:20:59 ⚠️  SQNC_IPFS IPFS: please run: 'ipfs init'
15:20:59 ⚠️  SQNC_IPFS IPFS:
15:20:59 🚨 SQNC_IPFS IPFS process unexpectedly exited with code 1

```

run:

```sh
# cd into where ipfs is

> cd node_modules/kubo/bin/ipfs
> # export it's path
> export IPFS_PATH=$PWD
> # cd to data folder (if this doesn't exist create it at root)
> ../node_modules/kubo/bin/ipfs init
> # then cd back to root and continue

Reviewer: ```suggestion
mkdir data && cd data/
# export it's path
export IPFS_PATH=$PWD
../node_modules/kubo/bin/ipfs init
# then cd back to root and continue
```
